supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJob.scala

```diff
@@ -176,7 +176,10 @@ final case class AwsBatchJob(jobDescriptor: BackendJobDescriptor, // WDL/CWL
     val jobDefinitionBuilder = StandardAwsBatchJobDefinitionBuilder
     val jobDefinitionContext = AwsBatchJobDefinitionContext(runtimeAttributes,
                                                             taskId,
-                                                            reconfiguredScript,
+                                                            awsBatchAttributes.fileSystem match {
```

Comment:
I think I'd prefer this `match` to be moved outside the `Constructor()` into its own value too

Reply:
Done.